contracts/src/v0.8/dev/automation/2_1/KeeperRegistryLogicB2_1.sol

```diff
@@ -353,7 +353,24 @@ contract KeeperRegistryLogicB2_1 is KeeperRegistryBase2_1 {
    * @notice calculates the minimum balance required for an upkeep to remain eligible
    * @param id the upkeep id to calculate minimum balance for
    */
-  function getMinBalanceForUpkeep(uint256 id) external view returns (uint96 minBalance) {
+  function getBalance(uint256 id) external view returns (uint96 balance) {
+    return s_upkeep[id].balance;
+  }
+
+  /**
+   * @notice calculates the minimum balance required for an upkeep to remain eligible
+   * @param id the upkeep id to calculate minimum balance for
+   */
+  function getMinBalance(uint256 id) external view returns (uint96) {
+    return getMinBalanceForUpkeep(id);
+  }
```

Comment:
needed to satisfy the `iAutomationRegistryConsumer` interface (mismatch caught by new test :tada:)